src/Compute/Compute.Test/ScenarioTests/ResourceSkuTests.ps1

```diff
@@ -25,3 +25,13 @@ function Test-GetResourceSku
 	Assert-True { $output.Contains("virtualMachines"); }
 	Assert-True { $output.Contains("Zones"); }
 }
+
+<#
+.SYNOPSIS
+Test List Resource Skus
```

Comment:
The synopsis is the same as the first test.

Change the synopsis to "Test List Resource Skus by Location."